script-base.js

```diff
@@ -14,7 +14,12 @@ var Generator = module.exports = function Generator() {
     this.appname = path.basename(process.cwd());
   }
   this.appname = this._.slugify(this._.humanize(this.appname));
-  this.scriptAppName = this._.camelize(this.appname) + angularUtils.appName(this);
+
+  try {
```

Comment:
change this try catch to make the `bower.json` variables available as a top level variable so it can be used on line 27(32) as well without having two try catch blocks for essentially the same thing.

eventually (soon) these are going int he .yo-rc.json file.